api/models/reservation.py

```diff
@@ -0,0 +1,31 @@
+"""Payment model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .order import Order
+from .workshopPrice import WorkshopPrice
+
+
+class Reservation(Base):
+    """Stores workshop reservations."""
+
+    workshop_price = models.ForeignKey(
+        WorkshopPrice,
+        verbose_name=_("Workshop price"),
+    )
+    order = models.ForeignKey(
+        Order,
+        verbose_name=_("Order"),
+    )
+    ends_at = models.DateTimeField(
```

Comment:
I have added Years REST endpoint in #55 to demonstrate the field<->json parameter translation. The translation must of course be set manually.

I think, that it is easy enough.